Test.Microsoft.NET.Build.Containers.Filesystem/TargetsTests.cs

```diff
@@ -89,12 +92,33 @@ public void CanSetEntrypointArgsToUseAppHost(bool useAppHost, params string[] en
     [TestMethod]
     public void CanNormalizeInputContainerNames(string projectName, string expectedContainerImageName, bool shouldPass)
     {
-        var project = InitProject(new()
+        var (project, dispose) = InitProject(new()
         {
             ["AssemblyName"] = projectName
         });
+        using var _ = dispose;
         var instance = project.CreateProjectInstance(global::Microsoft.Build.Execution.ProjectInstanceSettings.None);
         Assert.AreEqual(shouldPass, instance.Build(new[]{"ComputeContainerConfig"}, null, null, out var outputs), "Build should have succeeded");
         Assert.AreEqual(expectedContainerImageName, instance.GetPropertyValue("ContainerImageName"));
     }
+
+    [DataRow("7.0.100", true)]
+    [DataRow("8.0.100", true)]
+    [DataRow("7.0.100-preview.7", true)]
+    [DataRow("7.0.100-rc.1", true)]
+    [DataRow("6.0.100", false)]
+    [DataRow("7.0.100-preview.1", false)]
+    [TestMethod]
+    public void CanWarnOnInvalidSDKVersions(string sdkVersion, bool isAllowed) {
+        var (project, dispose) = InitProject(new()
+        {
+            ["NETCoreSdkVersion"] = sdkVersion,
+            ["PublishProfile"] = "DefaultContainer"
+        }, $"version-test-{sdkVersion}");
+        using var _ = dispose;
+        var instance = project.CreateProjectInstance(global::Microsoft.Build.Execution.ProjectInstanceSettings.None);
+        var derivedIsAllowed = Boolean.Parse(project.GetProperty("_IsSDKContainerAllowedVersion").EvaluatedValue);
```

Comment:
this explicit property extraction is safe because in the targets we ensure the property is always set